lib/openxml/elements/shape_non_visual.rb

```diff
@@ -0,0 +1,8 @@
+module OpenXml
+  module Elements
```

Comment:
Looks like it's also happening in a few other files. I wouldn't worry about it too much for the `a:` namespaced elements since we're already planning on extracting them, but for the ones in the PresentationML `p:` namespace, we should probably make sure they're namespaced under Pptx.